src/Widgets/DataWidget/DataBatchLoadingIndicator.tsx

```diff
@@ -8,7 +8,7 @@ export const DataBatchLoadingIndicator = connect(
     const dataScope = useData()
     const { hasMore } = useContext(DataBatchContext)
 
-    // use side-effects to trigger loading
+    // subscribe to dataScope.take() and hasMore() to trigger Loading
```

Comment:
* [ ] FYI the "to trigger" part of the comment is still misleading. What we do here is more of "subscribe to dataScope.take() and hasMore() loading".